omnigibson/utils/teleop_utils.py

```diff
@@ -0,0 +1,540 @@
+import carb
+import numpy as np
+import time
+from importlib import import_module
+from threading import Thread
+from typing import Iterable, Optional, Tuple
+
+import omnigibson as og
+import omnigibson.utils.transform_utils as T
+from omnigibson.objects import USDObject
+from omnigibson.robots import BaseRobot, LocomotionRobot
+
+
+class TeleopSystem():
```

Comment:
rubber-stamping this functionality for now, since I can't test it -- do we have unit tests for the VR system?

Reply:
second the idea of adding some unit tests.

Reply:
I think I can add some unit tests for the TeleopSystem base class, but won't be able for specific subclasses since they require additional hardware/library support that's not in the OG setup

Reply:
mock input

Reply:
added